### PR TITLE
Add 4.0 development post

### PR DESCRIPTION
**Pull request by oe**
This should be merged after 3.8 is released and all neccessary requirements have been done. Feel free to suggest improvements!

### CODE REVIEW

File: docs/_posts/2018-03-14-development-update.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Jekyll 4.0 is on the Horizon!"
+date: "2018-03-14 16:07:00 +0100"
+author: oe
+categories: [community]
+---
+
+With the release of Jekyll 3.8.0, it's been 2 and a half years since the last major release. Jekyll 3.0.0 was released in late October of 2015! That's a long time ago, and we've been working towards the next major release of Jekyll for a couple of months now. Here's a small preview of what's to come:
+
+- Dropping support for Ruby 2.1 and 2.2. Both versions are reaching or have reached their EOL period.
```
Comment (ashmaroli) [inline docs/_posts/2018-03-14-development-update.md]: Ruby 2.2 has officially reached EOL.. so the `reaching or..` can be removed..

File: docs/_posts/2018-03-14-development-update.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Dropping Pygments as a dependency. We're already defaulting to Rouge, and this removes the implicit Python dependency. (finally!)
+- Making the `link` tag use relative URLs. This is a big breaking change, but it's the cleaner solution.
+
+We're open to more ideas! If the development cost isn't too high, or if someone volunteers to take care of the implementation, it's likely that your suggestion might make it into Jekyll 4.0. Head over to this [issue] for more details. Some interesting topics might be improving Internationalization support in Jekyll, creating convenience Liquid tags, et cetera.
```
Comment (ashmaroli) [inline docs/_posts/2018-03-14-development-update.md]: link for `[issue]` needs fixing..

File: docs/_posts/2018-03-14-development-update.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+That being said, the development period of version 4.0 begins _now_. This means a couple of things:
+
+- New features will only be implemented in Jekyll 4.0. There will be no 3.9.0 or the like.
+- Same with bug fixes, unless they concern something introduced in Jekyll 3.7 or 3.8, in which case we will backport the fixes and release a patch version.
+- Now is a great time to finally take on the feature you've wanted to see in Jekyll for ages! Just open an issue or experiment with the code to get going!
+
+As for a release date, we're currently aiming for late summer, around September or so. However, keep in mind that this project is purely volunteer-run, and as such, delays might occur and we might not hit that release date.
+
+Finally, this is a great time for newcomers to open-source to make their first contribution! We'll be doing our best to mark recommended contributions and create newcomer-friendly issues, as well as to provide mentoring through the contribution process (although we'd like to think that we're already pretty proficient at that). So if you've always been hestitant about contributing to a large open-source project, Jekyll is a good place to start!
```
Comment (ashmaroli) [inline docs/_posts/2018-03-14-development-update.md]: /s `mentoring throughout the ..` perhaps?

File: docs/_posts/2018-03-14-development-update.md (hunk 1 above)
Comment (oe) [reply docs/_posts/2018-03-14-development-update.md]: :+1:

File: docs/_posts/2018-03-14-development-update.md (hunk 2 above)
Comment (oe) [reply docs/_posts/2018-03-14-development-update.md]: was planning to do that once we have the actual issue hehe

Comment (ashmaroli) [reply docs/_posts/2018-03-14-development-update.md]: :+1:

File: docs/_posts/2018-03-14-development-update.md (hunk 3 above)
Comment (DirtyF) [reply docs/_posts/2018-03-14-development-update.md]: Where does that estimation comes from? I'd rather ship on quality, than on date.

**Comment by oe** [comment]
@jekyllbot: merge +doc

**Comment by ashmaroli** [comment]
@oe, Wouldn't it be better if the date and filename were updated to the actual date of publishing..? (No issues.., just a nit..)

**Comment by oe** [comment]
@ashmaroli oh whoops, forgot about that

**Pull request merged**